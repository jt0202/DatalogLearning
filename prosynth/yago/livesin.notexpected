david_vickers	bangalore
charles_cusimano	chengdu
jeffrey_mcclain	gloucestershire
patricia_lantz	reigate
siva_selliah	kes√§ranta
