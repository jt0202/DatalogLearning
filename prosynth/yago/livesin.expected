rowan_williams	london
rowan_williams	canterbury
jean_chrétien	quebec
president_of_the_peoples_republic_of_china	zhongnanhai
john_sergeant_journalist	london
john_sergeant_journalist	ealing
john_manley	ottawa
gilles_duceppe	canada
gilles_duceppe	quebec
gilles_duceppe	montreal
roger_federer	wollerau
james_packer	bellevue_hill_new_south_wales
mike_huckabee	north_little_rock_arkansas
alan_sokal	nicaragua
peter_short_clergyman	fredericton
john_ensign	nevada
john_ensign	las_vegas_nevada
terry_everett	enterprise_alabama
tom_allen	portland_maine
w_pete_cunningham	charlotte_north_carolina
robin_morgan	new_york_city
robin_morgan	new_york
jean_lapierre	montreal
donald_c_macdonald	ontario
donald_c_macdonald	toronto
john_kline_politician	lakeville_minnesota
emperor_of_austria	vienna
julia_gillard	altona_victoria
lucienne_robillard	quebec
lucienne_robillard	montreal
henry_dworshak	burley_idaho
peter_atkins	oxford
richard_burr	winstonsalem_north_carolina
guillermo_vilas	buenos_aires
robert_kagan	virginia
tom_coburn	muskogee_oklahoma
john_blake_rice	chicago
fran_fine	manhattan
fran_fine	queens
dale_e_wolf	delaware
george_s_nixon	winnemucca_nevada
michael_kay_sports_broadcaster	new_york
john_hopkins_lawyer	united_kingdom
john_hopkins_lawyer	cambridge
lieutenant_governor_of_new_brunswick	fredericton
lieutenant_governor_of_new_brunswick	old_government_house_fredericton
ruby_dhalla	ontario
ruby_dhalla	mississauga
brian_masse	windsor_ontario
jake_siewert	new_york
john_c_hull	canada
andré_rouvoet	woerden
bill_saunderson	ontario
bill_saunderson	toronto
choi_kyungju	southlake_texas
ian_wilmut	scotland
andrew_breitbart	westwood_california
james_p_pope	boise_idaho
thomas_levet	palm_beach_gardens_florida
mike_beebe	arkansas
mike_beebe	searcy_arkansas
paul_crête	rivièreduloup
wendy_turnbull	boca_raton_florida
wendy_turnbull	sandgate_queensland
jonathan_tasini	new_york_city
jonathan_tasini	new_york
michael_gazzaniga	santa_barbara_california
jim_risch	boise_idaho
eilat_mazar	israel
daniel_rodney	delaware
daniel_rodney	lewes_delaware
agustín_calleri	buenos_aires
harold_bernard_st_john	ilaro_court
richard_r_kenney	delaware
richard_r_kenney	dover_delaware
james_hendler	usa
christian_bergmann	rostock
christian_bergmann	göttingen
mar_eshai_shimun_xxiii	chicago
mar_eshai_shimun_xxiii	illinois
odoardo_di_santo	north_york_ontario
yan_zi	sichuan
joyce_wethered	knightshayes_court
gunning_bedford	lt_SrDelawaregt
gunning_bedford	lt_SrNew_Castle_Delawaregt
wong_yan_lung	hong_kong
joseph_haslet	delaware
joseph_haslet	sussex_county_delaware
yoshie_onda	canada
james_j_greco	cheshire_connecticut
thomas_andrew_knight	herefordshire
jose_de_venecia	lt_JrPangasinangt
gerry_lopez	bend_oregon
eli_m_saulsbury	dover_delaware
barbara_fusarpoli	italy
barbara_fusarpoli	sesto_san_giovanni
ken_cheuvront	phoenix_arizona
beth_moore	texas
josé_acasuso	buenos_aires
satoshi_kataoka	japan
satoshi_kataoka	tokyo
melissa_gregory	newark_delaware
stefan_koubek	vienna
william_peters_hepburn	clarinda_iowa
arlene_blum	berkeley_california
björn_phau	weilerswist
li_jie_go_player	virginia
taiki_seto	japan
geoffrey_rowell	worth_west_sussex
martin_n_johnson	fargo_north_dakota
carlos_berlocq	buenos_aires
john_m_haines	boise_idaho
denis_pelli	new_york
mark_inglis	new_zealand
mark_inglis	hanmer_springs
kathleen_falk	madison_wisconsin
mark_foster_golfer	england
chris_hanell	scottsdale_arizona
jerry_anderson_golfer	canada
jerry_anderson_golfer	ontario
pip_elson	gloucestershire
guy_hunt_golfer	bournemouth
dennis_keene	wilder_kentucky
andrew_halcro	alaska
janne_holmén	uppsala
janne_holmén	sweden
pedro_sauer	leesburg_virginia
pedro_sauer	virginia
jagapati_babu	andhra_pradesh
jagapati_babu	india
margaret_sheffield	new_york
savannah_guthrie	washington_dc
murphy_jensen	atlanta
brian_moran	alexandria_virginia
jan_herman_linge	norway
yasuro_kikuchi	japan
eric_schneiderman	manhattan
eric_schneiderman	new_york_city
douglas_devananda	sri_lanka
douglas_devananda	colombo
srikumar_banerjee	india
srikumar_banerjee	new_delhi
william_sorrell	burlington_vermont
hans_von_pechmann	germany
sanjay_kirloskar	pune
aaron_schock	illinois
aaron_schock	peoria_illinois
andrew_fire	stanford_california
jean_case	mclean_virginia
raymond_buckley	manchester_new_hampshire
paul_sheehan_golfer	melbourne
scott_suder	abbotsford_wisconsin
michele_cantu	mexico
drew_meekins	colorado_springs_colorado
ben_buckley	sydney
john_coughlin_figure_skater	colorado_springs_colorado
omer_fattah_hussain	arbil
martin_wharton	gosforth
andy_welti	plainview_minnesota
edward_rondthaler	new_york
gurbanguly_berdimuhamedow	turkmenistan
jowilfried_tsonga	gingins
rebecca_twigley	melbourne
kevin_t_vantassell	utah
kevin_t_vantassell	vernal_utah
wes_shoemyer	clarence_missouri
rufus_hardy_representative	corsicana_texas
xavier_montsalvatge	barcelona
tony_hayward	england
tony_hayward	kent
eilley_bowers	washoe_county_nevada
martin_camirand	montreal
andriy_shevchenko_politician	ukraine
richard_thomas_lawyer	surrey
george_h_utter	rhode_island
george_h_utter	westerly_rhode_island
brian_bateman	monroe_louisiana
pelle_edberg	sweden
dick_day	owatonna_minnesota
gil_stratton	california
ričardas_berankis	lithuania
ričardas_berankis	vilnius
sharon_ito	sacramento_california
thomas_j_mabry	clovis_new_mexico
roy_brown_montana_politician	billings_montana
regina_d_thomas	savannah_georgia
eddie_day_pashinski	wilkesbarre_pennsylvania
shayne_lewis	springfield_guiding_light
christine_scanlan	dillon_colorado
ivana_abramović	zagreb
joseph_warder	jeffersonville_indiana
shaz_granger	london
james_durbin	united_kingdom
richard_lestock	united_kingdom
mark_brown_golfer	new_zealand
mark_brown_golfer	lower_hutt
bill_vasey	rawlins_wyoming
alison_redford	alberta
patricia_lantz	gig_harbor_washington
sylvi_listhaug	oslo
vincent_j_polistina	egg_harbor_township_new_jersey
patricia_gras	texas
charles_a_doswell_iii	norman_oklahoma
jay_goyal	mansfield_ohio
andrew_boff	london_borough_of_hackney
william_thorsell	ontario
william_thorsell	toronto
william_thorsell	mulmur_ontario
harry_jayawardena	colombo
a_george_pradel	naperville_illinois
steve_street	revillo_south_dakota
edgar_mcgraw	texas
sima_nikolic	shreveport
charles_f_brannock	syracuse_new_york
enrique_caballero_peraza	acapulco
enrique_caballero_peraza	guerrero
george_william_alexander	stoke_newington
julie_mennella	pennsylvania
jam_sadiq_ali	jam_nawaz_ali
anastasia_pivovarova	barcelona
anastasia_pivovarova	spain
pierredominique_bazaine	russian_empire
matt_szollosi	oregon_ohio
leta_lindley	palm_beach_gardens_florida
shaukat_tarin	islamabad
phil_sterner	rosemount_minnesota
grace_diaz	providence_rhode_island
georg_balthasar_metzger	germany
paul_pelosi	lt_JrCaliforniagt
paul_pelosi	lt_JrSan_Franciscogt
kamil_čapkovic	bratislava
mary_ann_coady_weinand	tucson_arizona
alexander_john_scott	united_kingdom
selvam_adaikalanathan	sri_lanka
george_mason_vi	gunston_hall
michael_sim	perth_western_australia
michael_sim	western_australia
brian_lenihan	lt_JnrCastleknockgt
liza_andriyani	jakarta
danny_martiny	louisiana
danny_martiny	kenner_louisiana
masud_ahmad	islamabad_capital_territory
subir_sachdev	cambridge_massachusetts
justin_hobday	pretoria
justin_hobday	south_africa
nico_van_rensburg	pretoria
nico_van_rensburg	south_africa
kyle_johansen	ketchikan_alaska
tomoko_namba	japan
tomoko_namba	niigata_niigata
tig_trager	charming_california
albert_a_bühlmann	switzerland
albert_a_bühlmann	zürich
david_b_rivkin	washington_dc
james_ryan_actor	new_york
henderson_jordan_louisiana_sheriff	arcadia_louisiana
shinzo_machida	brazil
shinzo_machida	belém
petra_rampre	ltŽirigt
christina_gao	canada
christina_gao	toronto
bob_krist	omaha_nebraska
siva_selliah	sri_lanka
siva_selliah	colombo
siva_selliah	cinnamon_gardens
tim_burgess_seattle	seattle
charles_cusimano	metairie_louisiana
andreas_demetriou	cyprus
lloyd_hulbert	manhattan_kansas
lucius_horatio_stockton	new_jersey
lucius_horatio_stockton	trenton_new_jersey
manny_villar	metro_manila
manny_villar	las_piñas_city
stephen_slesnick	canton_ohio
bill_heath_politician	bremen_georgia
iñigo_cervanteshuegun	hondarribia
bradford_stroh	california
julius_kiesner	milwaukee
marcus_r_clark	west_monroe_louisiana
mina_harigae	monterey_california
james_morrison_golfer	england
james_morrison_golfer	weybridge
william_e_dunn	murray_utah
roxy_olin	los_angeles
blaž_rola	ptuj
julien_guerrier	la_rochelle
els_demol	herent
jaffer_ali_entrepreneur	chicago
jaffer_ali_entrepreneur	illinois
jarod_scott	anaheim_california
jarod_scott	mount_juliet_tennessee
julia_kristeva	france
president_of_the_united_states	white_house
prime_minister_of_the_united_kingdom	england
prime_minister_of_the_united_kingdom	london
prime_minister_of_the_united_kingdom	lt10_Downing_Streetgt
president_of_france	paris
president_of_france	ltÉlysée_Palacegt
wilhelm_wundt	germany
robert_devereux	lt_2nd_Earl_of_EssexLondongt
robert_dudley	lt_1st_Earl_of_LeicesterEssexgt
robert_dudley	lt_1st_Earl_of_LeicesterKenilworth_Castlegt
robert_dudley	lt_1st_Earl_of_LeicesterLondongt
robert_dudley	lt_1st_Earl_of_LeicesterWarwickshiregt
john_dudley	lt_1st_Duke_of_NorthumberlandKenilworth_Castlegt
john_dudley	lt_1st_Duke_of_NorthumberlandWest_Midlands_countygt
john_dudley	lt_1st_Duke_of_NorthumberlandWarwick_Castlegt
john_dudley	lt_1st_Duke_of_NorthumberlandWarwickshiregt
john_dudley	lt_1st_Duke_of_NorthumberlandSyon_Housegt
prime_minister_of_finland	kesäranta
leader_of_the_official_opposition_canada	stornoway_residence
henry_percy	lt_2nd_Earl_of_NorthumberlandWarkworth_Castlegt
axel_thue	norway
tad_martin	pine_valley_pennsylvania
ludwig_prandtl	germany
richard_evans_schultes	cambridge
edward_forbes	united_kingdom
mildred_dresselhaus	united_kingdom
maria_augimeri	ontario
prime_minister_of_moldova	chişinău
lorraine_hunt	nevada
lorraine_hunt	las_vegas_nevada
rahul_mahajan_reality_tv_personality	india
rahul_mahajan_reality_tv_personality	mumbai
mark_ricks	rexburg_idaho
alex_rider_character	england
olivia_winters	wisconsin
matt_freeman_power_of_five	england
matt_freeman_power_of_five	york
john_keill	england
penelope_wensley	government_house_brisbane
john_grubb_delaware_settler	brandywine_hundred
einstein_family	germany
einstein_family	italy
einstein_family	switzerland
camil_mureşanu	romania
alan_baker_mathematician	united_kingdom
johann_georg_macasius	germany
anita_de_braganza	newport_rhode_island
riazuddin_physicist	islamabad_capital_territory
riazuddin_physicist	islamabad
denise_ilitch	michigan
gary_goldschneider	amsterdam
gary_goldschneider	netherlands
cynthia_villar	metro_manila
cynthia_villar	las_piñas_city
first_lady_of_the_united_states	white_house
justus_von_liebig	grand_duchy_of_hesse
chancellor_of_the_exchequer	lt11_Downing_Streetgt
victoria_lord	llanview_pennsylvania
jessica_buchanan	llanview_pennsylvania
nebetah	thebes_egypt
david_vickers	llanview_pennsylvania
henuttaneb	thebes_egypt
iset_daughter_of_amenhotep_iii	thebes_egypt
thomson_mason	chopawamsic_plantation
ross_rayburn	french_polynesia
danielle_rayburn	llanview_pennsylvania
salim_mehmud	sindh
